pages/guide.md

```diff
@@ -63,7 +64,14 @@ BOD 18-01 also mandates two additional steps:
 
 [Preloading a domain](https://https.cio.gov/hsts/#hsts-preloading) enforces the use of HTTPS across an entire zone, and is technical compliance with the HTTPS usage requirements of BOD 18-01. Preloading allows agencies to avoid inventorying and configuring an HSTS policy for every individual subdomain, though this necessarily impacts all subdomains present on the domain, including intranet subdomains. Thus, all subdomains will need to support HTTPS in order to remain reachable for use in major browsers. Even with these two obstructions, preloading a domain can be a reality with coordinated effort.
 
-Second-level .gov domains that are [only used to redirect visitors to other websites](https://https.cio.gov/guide/#what-about-domains-that-are-only-used-to-redirect-visitors-to-other-websites%3f) and are not used on intranets are excellent preloading candidates. However, DHS **strongly recommends** that federal agencies perform a thorough evaluation of those domains that are highly trafficked or otherwise have significant value. Those are likely to be the domains that citizens and intra-agency users stand to benefit most from the always-HTTPS approach that preloading provides.
+Second-level .gov domains that are [only used to redirect visitors to other websites](https://https.cio.gov/guide/#what-about-domains-that-are-only-used-to-redirect-visitors-to-other-websites) and are not used on intranets are excellent preloading candidates. However, DHS **strongly recommends** that federal agencies perform a thorough evaluation of those domains that are highly trafficked or otherwise have significant value. Those are likely to be the domains that citizens and intra-agency users stand to benefit most from the always-HTTPS approach that preloading provides.
+
+#### My device is HTTPS already and doesn't natively support HSTS. Can DHS mark this a false positive?
+HSTS is intended to protect users, not servers. Even if a device is HTTPS-only, for as long as HTTP remains the default protocol for browsers, users are at risk from the [insecure redirect](https://https.cio.gov/hsts/#background). A lack of HSTS means that an attacker with an on-path vantage point can redirect or attempt to spoof a government web service.
```

Comment:
> Even if a device is HTTPS-only

Recommend "Even if a device is HTTPS-only and doesn't respond over port 80" to really hammer this home.

> for as long as HTTP remains the default protocol for browsers

I recommend "browsers and other HTTP clients".

> users are at risk from the insecure redirect

Recommend "users are at risk from insecure redirects".

> A lack of HSTS means that an attacker with an on-path vantage point can redirect or attempt to spoof a government web service.

Recommend "Without HSTS, clients may still issue HTTP requests even when the server does not support them, and an attacker with an on-path vantage point can redirect or spoof these requests for malicious purposes."